mis_builder/models/mis_builder.py

```diff
@@ -119,6 +131,50 @@ def _onchange_name(self):
                 }
             }
 
+    @api.multi
+    def _compute_expression(self):
+        for kpi in self:
+            kpi.expression = ''
+            for expression in kpi.expression_ids:
+                if expression.subkpi_id:
+                    kpi.expression += '%s :\n' % expression.subkpi_id.name
+                kpi.expression += '%s\n' % expression.name
+
+    @api.multi
+    def _inverse_expression(self):
+        for kpi in self:
+            if kpi.multi:
+                raise UserError('Can not update a multi kpi from the kpi line')
```

Comment:
Spelling: Both cannot and can not are acceptable spellings, but the first is much more usual. http://www.oxforddictionaries.com/words/cannot-or-can-not